NEWS.md

```diff
@@ -1,3 +1,11 @@
+### v9.8.1 (2023-01-25)
+
+* Changed GCP metadata parsing to use json-bigint to avoid loss of precision from numerical instance ID.
+
+* Fixed dead links in the docs.

```

Comment:
Not sure if this is worthy of a note

Reply:
I can pull it. It wasn't my commit, so I didn't want to stomp on it.

Reply:
(Stomp)